packages/components/psammead-most-read/README.md

```diff
@@ -15,6 +15,45 @@ The `MostRead` component is designed to display the most read articles given a d
 
 ## Components (WIP)
 
+## MostRead
+### Props
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| items | array | yes | N/A | `[{ title: 'This is a headline', href: 'https://www.bbc.com' }]`
+| header | string | yes | N/A | `'Most Read'` |
+| service | string | yes | N/A | `'news'` |
+| script | object | yes | N/A | `{ canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }` |
+| dir | string | yes | 'ltr' | `'ltr'`|
+
+### Usage
+
+```jsx
+import React from 'react';
+import MostRead from '@bbc/psammead-most-read;
+import { latin } from '@bbc/gel-foundations/scripts'
+
+const itemsLTR = [
```

Comment:
It would be good to name this something that'd be clear for someone who is new to this component. 
```suggestion
const items = [
```
And to make the titles/hrefs actually related to one another, because currently the News John Lewis story is linking to a vietnamese page.